CONTRIBUTING.md

```diff
@@ -82,6 +84,16 @@ go test ./internal/command/...
 go test ./internal/addrs
 ```
 
+### Build with Docker
+
+Install [Docker](https://docs.docker.com/engine/install/) and, if you're on Linux, run the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/).
+
+```
+docker run --rm -v "$PWD":/usr/src/opentofu -w /usr/src/opentofu golang:1.20.7 go build -v -buildvcs=false .
```

Comment:
If this is not on linux, then I believe you'll also want to pass GOOS and GOARCH explicitly. I think it's worth adding this here. So
```
docker run --rm -v "$PWD":/usr/src/opentofu -w /usr/src/opentofu golang:1.20.7 GOOS=linux GOARCH=amd64 go build -v -buildvcs=false .
```